airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```
@@ -4777,6 +4775,12 @@ interpolation:
     - title: stream_slice
       description: This variable is deprecated. Use stream_interval or stream_partition instead.
       type: object
+    - title: components_values
+      description: The record object produced by the components resolver for which a stream will be generated.
+      type: object
+      examples:
+        - name: "accounts"
+          id: 1234  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)